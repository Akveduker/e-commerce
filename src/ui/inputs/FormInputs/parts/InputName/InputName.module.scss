@import "../../../../../styles/_mixins.scss";
.name {
    @include authorizationLabelText;
}
