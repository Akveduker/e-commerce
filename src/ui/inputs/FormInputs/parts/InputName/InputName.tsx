import React, { FC } from 'react';
import s from './InputName.module.scss'
interface InputNameProps {
    inputName: string;
}
const InputName: FC<InputNameProps> = ({ inputName }) => {
    return (
        <p className={s['name']}>{inputName}</p>
    );
};

export default InputName;