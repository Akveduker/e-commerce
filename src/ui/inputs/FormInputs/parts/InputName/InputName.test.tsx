import { render, screen } from "@testing-library/react"
import InputName from "./InputName"

describe('InputName', () => {
    test('Рендерится имя, снапшот', () => {
        const { container } = render(<InputName inputName="name" />)
        expect(screen.queryByText('name')).toBeInTheDocument()
        expect(container).toMatchSnapshot()
    })
})