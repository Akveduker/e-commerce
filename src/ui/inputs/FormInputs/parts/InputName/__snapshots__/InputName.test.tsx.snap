// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputName Рендерится имя, снапшот 1`] = `
<div>
  <p
    class="name"
  >
    name
  </p>
</div>
`;
